cmd/reconciler/ip.go

```diff
@@ -28,21 +63,22 @@ func main() {
 	}
 	if err != nil {
 		_ = logging.Errorf("failed to create the reconcile looper: %v", err)
```

Comment:
The `_ = logging.Error()` syntax might be throwing me off... but I think this still causes an output logging print -- and just throws away the return value.

In the case that we tolerate a known error, do we want to log (and therefore print) the error twice (once to state the error, and a second time to say it's a known error)? Alternatively, we could remove this line and move it to the `else` condition of `tolerateKnownErrorCasesOrFail`, which would mean we always print the error once.

Reply:
Solidly done. I went ahead and made it so that the error message is passed in as a parameter to the `tolerateKnownErrorCasesOrFail()` method, and now it should only be logged once-per-each instead of a double logging in the case of a known error